speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/Mediator.h

```diff
@@ -28,9 +29,12 @@ class Mediator
 
     void SetBlackBoxFilePath(const char* p_filePath);
 
+    void SetSyncOption(bool p_sync);
```

Comment:
```suggestion
    void SetBlackBoxSyncOption(bool p_sync);
```

Reply:
I'm fine with changing this name, however as it is, the current name reflects the already existing {Set/Get}SyncOption in SDAConfig. In which case, we should be consistent.